README.md

```diff
@@ -1,56 +1,71 @@
-
-## Prerequisites
-
-* [golang](https://golang.org/doc/install)
-
-## Getting Started
-
-- See the [SDK Guide](docs/guide.md)
+# Cosmos SDK
+![banner](docs/graphics/cosmos-sdk-image.png)
+
+[![version](https://img.shields.io/github/tag/cosmos/cosmos-sdk.svg)](https://github.com/cosmos/cosmos-sdk/releases/latest)
+[![API Reference](https://godoc.org/github.com/cosmos/cosmos-sdk?status.svg
+)](https://godoc.org/github.com/cosmos/cosmos-sdk)
+[![Rocket.Chat](https://demo.rocket.chat/images/join-chat.svg)](https://cosmos.rocket.chat/)
+[![license](https://img.shields.io/github/license/cosmos/cosmos-sdk.svg)](https://github.com/cosmos/cosmos-sdk/blob/master/LICENSE)
+[![LoC](https://tokei.rs/b1/github/cosmos/cosmos-sdk)](https://github.com/cosmos/cosmos-sdk)
+[![Go Report Card](https://goreportcard.com/badge/github.com/cosmos/cosmos-sdk)](https://goreportcard.com/report/github.com/cosmos/cosmos-sdk)
+
+Branch    | Tests | Coverage
+----------|-------|---------
+develop   | [![CircleCI](https://circleci.com/gh/cosmos/cosmos-sdk/tree/develop.svg?style=shield)](https://circleci.com/gh/cosmos/cosmos-sdk/tree/develop) | [![codecov](https://codecov.io/gh/cosmos/cosmos-sdk/branch/develop/graph/badge.svg)](https://codecov.io/gh/cosmos/cosmos-sdk)
+master    | [![CircleCI](https://circleci.com/gh/cosmos/cosmos-sdk/tree/master.svg?style=shield)](https://circleci.com/gh/cosmos/cosmos-sdk/tree/master) | [![codecov](https://codecov.io/gh/cosmos/cosmos-sdk/branch/master/graph/badge.svg)](https://codecov.io/gh/cosmos/cosmos-sdk)
+
+**WARNING**: the libraries are still undergoing breaking changes as we get better ideas and start building out the Apps.
+
+**Note**: Requires [Go 1.9+](https://golang.org/dl/)
+
+
+## Overview
+
+The Cosmos SDK is a platform for building multi-asset Proof-of-Stake
+cryptocurrencies, like the [Cosmos Hub](https://cosmos.network). It is both a library for building applications and a set of tools for securely interacting with them.
+
+The goal of the Cosmos-SDK is to allow developers to easily create their custom interoperable blockchain applications within the Cosmos Network without having to code every single functionality of their application, thus abstracting away the complexities of building a Tendermint ABCI application. We envision the SDK as the `npm`-like framework to build secure blockchain applications on top of Tendermint.
+
+In terms of its design, the SDK optimizes flexibility and security. The framework is designed around a modular execution stack which allows applications to mix and match elements as desired. In addition, all modules are sandboxed for greater application security.
+
+It is based on two major principles:
+
+- **Composability**: Anyone can create a module for the Cosmos-SDK, and using the already-built modules in your blockchain is as simple as importing them into your application.  As a developer, you only have to create the modules required by your application that do not already exist.

```

Comment:
first sentence is good but I had to read it twice to understand the content. I'd change `and using` -> `and integrating`

Reply:
> As a developer, you only have to create the modules required by your application that do not already exist

Isn't that obvious? I'd remove that sentence all together

Reply:
I don't think that's obvious and is worth mentioning.